### PR TITLE
New Advances Design

### PR DESCRIPTION
**Pull request by siddharth0-0**
image

image

image

### CODE REVIEW

File: src/app/fyle/my-advances/my-advances-card/my-advances-card.component.scss [hunk 1]
```diff
@@ -5,50 +5,75 @@ $advance_pending_approval_color: #1ab6bb;
 $advance_approved_color: #66bb6a;
 $advance_paid_color: darken(#5cb85c, 15%);
 $advance_rejected_color: #d50000;
+@import '../../../../theme/colors.scss';
```
Comment (Dimple16) [inline src/app/fyle/my-advances/my-advances-card/my-advances-card.component.scss:8]: import at the top

File: src/app/fyle/my-advances/my-advances-card/my-advances-card.component.scss [hunk 2]
```diff
@@ -5,50 +5,75 @@ $advance_pending_approval_color: #1ab6bb;
...
   }
 
-  &--body {
-    margin: 8px 16px;
-    padding: 16px;
-    border-radius: 4px;
-    box-shadow: 0 0 8px rgba(0, 0, 0, 0.1);
+  &--date {
+    color: $grey-light;
+    font-size: 14px;
```
Comment (Dimple16) [inline src/app/fyle/my-advances/my-advances-card/my-advances-card.component.scss:25]: check if 14px is by default

File: src/app/fyle/my-advances/my-advances-card/my-advances-card.component.scss [hunk 3]
```diff
@@ -5,50 +5,75 @@ $advance_pending_approval_color: #1ab6bb;
...
+  }
+
+  &--approval {
+    margin-bottom: 8px;
   }
 
   &--pill {
-    display: inline-block;
-    background: #2196f3;
     color: #fff;
```
Comment (Dimple16) [inline src/app/fyle/my-advances/my-advances-card/my-advances-card.component.scss:73]: use `$pure-white`

File: src/app/fyle/team-advance/team-adv-card/team-adv-card.component.html [hunk 4]
```diff
@@ -1,26 +1,29 @@
...
-    <div class="advance-card--info">
-      <div>
-        <span class="advance-card--currency" *ngIf="advanceRequest.areq_currency && advanceRequest.areq_amount">{{
-          advanceRequest.areq_currency
-        }}</span>
-        <span class="advance-card--amount" *ngIf="advanceRequest.areq_amount && advanceRequest.areq_currency">{{
-          advanceRequest.areq_amount | number: '1.2-2'
-        }}</span>
+<div *ngIf="showDate" class="advance-card--date">
+  {{ advanceRequest.areq_created_at | date: 'EEEE' }},{{ advanceRequest.areq_created_at | date: 'MMM dd, yyyy' }}
```
Comment (Dimple16) [inline src/app/fyle/team-advance/team-adv-card/team-adv-card.component.html:2]: same as above

File: src/app/fyle/team-advance/team-adv-card/team-adv-card.component.html [hunk 5]
```diff
@@ -1,26 +1,29 @@
...
+  </div>
 
-    <div class="advance-card--status">
-      <div *ngIf="advanceRequest.areq_approved_at">
-        Approved on {{ advanceRequest.areq_approved_at | date: 'MMM dd, yyyy' }}
+  <div class="advance-card--secondary-info">
+    <div class="advance-card--secondary-info-block">
+      <div class="advance-card--approval">
+        <div *ngIf="advanceRequest.areq_approved_at">
+          <span>Approved on {{ advanceRequest.areq_approved_at | date: 'MMM dd, yyyy' }}</span>
```
Comment (Dimple16) [inline src/app/fyle/team-advance/team-adv-card/team-adv-card.component.html:25]: can you add a screenshot with the approved on screen?

File: src/app/fyle/team-advance/team-adv-card/team-adv-card.component.scss [hunk 6]
```diff
@@ -5,49 +5,75 @@ $advance_pending_approval_color: #1ab6bb;
 $advance_approved_color: #66bb6a;
 $advance_paid_color: darken(#5cb85c, 15%);
 $advance_rejected_color: #d50000;
+@import '../../../../theme/colors.scss';
```
Comment (Dimple16) [inline src/app/fyle/team-advance/team-adv-card/team-adv-card.component.scss:8]: same as above

File: src/app/fyle/team-advance/team-adv-card/team-adv-card.component.ts [hunk 7]
```diff
@@ -10,15 +11,26 @@ import { AdvanceRequestService } from 'src/app/core/services/advance-request.ser
...
+  currencySymbol = '';
+
+  showDate = true;
+
   actionOpened = false;
 
   constructor(private advanceRequestService: AdvanceRequestService) {}
 
   ngOnInit() {
+    console.log('sam', this.advanceRequest);
```
Comment (Dimple16) [inline src/app/fyle/team-advance/team-adv-card/team-adv-card.component.ts:29]: remove

**Review by Dimple16 — CHANGES_REQUESTED** [review]
added comments

**Review by Dimple16 — APPROVED** [review]

File: src/app/fyle/team-advance/team-adv-card/team-adv-card.component.ts [hunk 8]
```diff
@@ -10,15 +11,25 @@ import { AdvanceRequestService } from 'src/app/core/services/advance-request.ser
...
+
+  showDate = true;
+
   actionOpened = false;
 
   constructor(private advanceRequestService: AdvanceRequestService) {}
 
   ngOnInit() {
+    this.showDate =
+      (this.advanceRequest && new Date(this.advanceRequest.areq_created_at).toDateString()) !==
```
Comment (Julias0) [inline src/app/fyle/team-advance/team-adv-card/team-adv-card.component.ts:30]: use moment to do this.

File: src/app/fyle/my-advances/my-advances-card/my-advances-card.component.ts [hunk 9]
```diff
@@ -10,14 +12,24 @@ import { AdvanceRequestService } from 'src/app/core/services/advance-request.ser
...
 
+  currencySymbol = '';
+
+  showDate = true;
+
   constructor(private advanceRequestService: AdvanceRequestService) {}
 
   ngOnInit() {
+    this.showDate =
+      (this.advanceRequest && moment(new Date(this.advanceRequest.created_at)).format('ddd MMM DD YYYY')) !==
```
Comment (Julias0) [inline src/app/fyle/my-advances/my-advances-card/my-advances-card.component.ts:30]: arrey na na use moment to compare the dates. Dont convert to string and compare

File: src/app/fyle/team-advance/team-adv-card/team-adv-card.component.ts [hunk 10]
```diff
@@ -10,15 +12,25 @@ import { AdvanceRequestService } from 'src/app/core/services/advance-request.ser
...
+  showDate = true;
+
   actionOpened = false;
 
   constructor(private advanceRequestService: AdvanceRequestService) {}
 
   ngOnInit() {
+    this.showDate =
+      (this.advanceRequest && moment(new Date(this.advanceRequest.areq_created_at)).format('ddd MMM DD YYYY')) !==
+      (this.prevDate && moment(new Date(this.prevDate)).format('ddd MMM DD YYYY'));
```
Comment (Julias0) [inline src/app/fyle/team-advance/team-adv-card/team-adv-card.component.ts:32]: same comment as above

**Review by Julias0 — APPROVED** [review]

**Pull request merged**